tmt/base.py

```diff
@@ -289,7 +289,17 @@ def export(self, format_='yaml', keys=None):
     def lint_keys(self, additional_keys):
         """ Return list of invalid keys used, empty when all good """
         known_keys = additional_keys + self._keys
-        return [key for key in self.node.get().keys() if key not in known_keys]
+        node_keys = list(self.node.get().keys())
+        prop_adjust = self.node.get('adjust')
+        if prop_adjust is not None:
+            if isinstance(prop_adjust, dict):
+                node_keys.extend(prop_adjust.keys())
+            elif isinstance(prop_adjust, list) or isinstance(prop_adjust, tuple):
```

Comment:
Second argument can be tuple of allowed types so you can use
```python
elif isinstance(prop_adjust, (list, tuple)):
```

Reply:
Updated, thanks!